pkg/cmd/cli/cmd/tag.go

```diff
@@ -133,7 +133,7 @@ func parseStreamName(defaultNamespace, name string) (string, string, error) {
 
 func determineSourceKind(f *clientcmd.Factory, input string) string {
 	mapper, _ := f.Object()
-	gvks, err := mapper.KindsFor(schema.GroupVersionResource{Group: imageapi.GroupName, Resource: input})
+	gvks, err := mapper.KindsFor(schema.GroupVersionResource{Group: imageapi.LegacyGroupName, Resource: input})
```

Comment:
@soltysh @sttts this is new in 1.6 right? I mean the `ResourcesShortcutStatic` does not exists prior to this PR... 
I think we have to add our groups there for the shortcuts to work properly.